modules/installation-osp-kuryr-settings.adoc

```diff
@@ -0,0 +1,175 @@
+----
+
+. Open the `cluster-network-03-config.yml` file in an editor and enter a CR that
+describes the Operator configuration you want:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec: <1>
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  serviceNetwork:
+  - 172.30.0.0/16
+  defaultNetwork:
+    type: Kuryr
+    kuryrConfig:
+      enablePortPoolsPrepopulation: false <2>
+      poolMinPorts: 1 <3>
+      poolBatchPorts: 3 <4>
+      poolMaxPorts: 5 <5>
+      openstackServiceNetwork: 172.30.0.0/15 <6>
+----
+<1> The parameters for the `spec` parameter are only an example. Specify your
+configuration for the Cluster Network Operator in the CR.
+<2> Set `enablePortPoolsPrepopulation` to `true` to make Kuryr proactively
+create new Neutron ports when a namespace is created or new node is added to
+the cluster. This will use more Neutron ports quota, but may speed up spawning
+pods. Defaults to `false`.
+<3> Kuryr will create new ports for a pool when number of free ones in that
+pool is lower than `poolMinPorts`. Defaults to `1`.
+<4> `poolBatchPorts` controls how many new ports are created when number of
+free ports is lower than `poolMinPorts`. Defaults to `3`.
+<5> When number of free ports in a pool is higher than `poolMaxPorts` Kuryr
+will delete them until the number is exactly `poolMaxPorts`. Setting `0`
+will disable this upper bound, effectively preventing pools from shrinking and
+this is the default value.
+<6> `openStackServiceNetwork` contains the CIDR of network from which to
+allocate IPs for OpenStack Octavia's LoadBalancers. Please note that with
+Amphora driver Octavia uses two IPs from that network for each loadbalancer -
+one given by OpenShift and second for VRRP connections. As the first one is
+managed by OpenShift's and second by Neutron's IPAMs, those need to come from
+different pools. Therefore, `openStackServiceNetwork` needs to be at least
+twice the size of `serviceNetwork`, and whole `serviceNetwork` must be
+overlapping with `openStackServiceNetwork`. cluster-network-operator will then
+make sure VRRP IPs are taken from the ranges inside `openStackServiceNetwork`
+that are not overlapping with `serviceNetwork`, effectivly preventing
+conflicts. If not set, cluster-network-operator will use `serviceNetwork`
+expanded by decrementing the prefix size by 1.
++
+The CNO provides default values for the parameters in the CR, so you must
+specify only the parameters that you want to change.
+
+. Save the `cluster-network-03-config.yml` file and quit the text editor.
+. Optional: Back up the `manifests/cluster-network-03-config.yml` file. The
+installation program deletes the `manifests/` directory when creating the
+cluster.
++
+. Proceed with the rest of installation normally:
++
+[source,terminal]
+----
+$ ./openshift-install create cluster --dir=<installation_directory>
+----
+
+.Modifying Kuryr options after the installation
```

Comment:
This will end up being a separate module, too. We can add it to a post-installation tasks assembly.